deploy.sh

```diff
@@ -0,0 +1,36 @@
+if ([ $TRAVIS_PULL_REQUEST == "false" ])
+then
+    #Clone the latest gh-pages
+    git clone https://github.com/ScottLogic/StockFlux.git --branch gh-pages gh-pages
+
+    #Get line with version from the file -> get the second word -> remove brackets
+    VERSION=$(grep "version" package.json | awk -v N=$2 '{print $2}' | cut -d \" -f2)
+
+    #Get line with the release type (develop/master) from the file -> get the second word -> remove brackets
```

Comment:
The remove brackets comment confused me for a while.. I think it should be "quotes".